control-plane-operator/main.go

```diff
@@ -154,7 +154,11 @@ func NewStartCommand() *cobra.Command {
 			setupLog.Error(err, "unable to detect cluster capabilities")
 			os.Exit(1)
 		}
-
+		activeImage, err := util.LookupActiveContainerImage(context.TODO(), kubeClient.CoreV1().Pods(namespace), os.Getenv("POD_NAME"), "control-plane-operator")
```

Comment:
You have access to `ctx` here, should probably use it

Reply:
Might even want to make a subcontext with a timeout

Reply:
done